L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -64,38 +64,65 @@
 _phase2_siml1emulator.add(hgcalTriggerPrimitivesTask)
 
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
-#phase2_hgcal.toReplaceWith( SimL1EmulatorTask , _phase2_siml1emulator )
-
 from Configuration.Eras.Modifier_phase2_hgcalV11_cff import phase2_hgcalV11
 (phase2_hgcal & ~phase2_hgcalV11).toReplaceWith( SimL1EmulatorTask, _phase2_siml1emulator )
 
-#%% # Barrel EGamma
-#%% # ########################################################################
+# Barrel EGamma
+# ########################################################################
 from L1Trigger.L1CaloTrigger.L1EGammaCrystalsEmulatorProducer_cfi import *
 _phase2_siml1emulator.add(L1EGammaClusterEmuProducer)
 
 from L1Trigger.L1CaloTrigger.l1EGammaEEProducer_cfi import *
 _phase2_siml1emulator.add(l1EGammaEEProducer)
 
+from Configuration.Eras.Modifier_phase2_trigger_cff import phase2_trigger
+phase2_trigger.toReplaceWith( SimL1EmulatorTask , _phase2_siml1emulator)
```

Comment:
Here you include `hgcalTriggerPrimitivesTask` ignoring `phase2_hgcalV11`

Reply:
hm.  you are right.
With the latest fix from Jean-Baptiste we should not be excluding the v11.
So the above check
`(phase2_hgcal & ~phase2_hgcalV11).toReplaceWith( SimL1EmulatorTask, _phase2_siml1emulator )`
should be removed, which serves no purpose, as it is now.

Reply:
I think we should still have the (currently commented-out) line:
`phase2_hgcal.toReplaceWith( SimL1EmulatorTask , _phase2_siml1emulator )`

and as above, we might want to separate the different contributions to this task somehow.

Reply:
agree.

Reply:
divided phase-2  modules in three tasks:  for TPs, L1T, and L1T-tracker-dependent.